Sources/Surge/Linear Algebra/Matrix.swift

```diff
@@ -963,17 +963,19 @@ public func eigenDecompose(_ lhs: Matrix<Double>) throws -> MatrixEigenDecomposi
     var leftEigenVectorWork = [Double](repeating: 0, count: Int(leftEigenVectorCount * matrixColCount))
     var rightEigenVectorWork = [Double](repeating: 0, count: Int(rightEigenVectorCount * matrixColCount))
 
-    let decompositionJobV = UnsafeMutablePointer<Int8>(mutating: ("V" as NSString).utf8String)
+    var decompositionJobV = "V".cString(using: .utf8) ?? []
+    var decompositionJobV2 = "V".cString(using: .utf8) ?? []
```

Comment:
Could we hardcode the C string as an array literal?

```suggestion
    var decompositionJobV: [CChar] = [0x56, 0x00] // "V" (compute)
    var decompositionJobV2: [CChar] = [0x56, 0x00] // "V" (compute)
```

Reply:
Also, while we're taking another look at this, what do y'all think about renaming these to `decompositionJobVL` and `decompositionJobVR`, respectively?

Reply:
Yes! 👍

Reply:
I thought the string version might be a touch more readable, but can switch to the array. Originally, the same variable `decompositionJobV` was used for both parameters, but two were needed for the new approach (hence V2), but I like your suggestion of L and R better. Will push an update with changes.

Reply:
@BrettThePark Thanks for making those changes! I agree that the string version is more readable, but for something so trivial, a compile-time constant feels more appropriate (honestly, I'm baffled as to why the LAPACK authors opted for "V" / "N" instead of a conventional Boolean).